docs/metrics.md

```diff
@@ -17,8 +17,8 @@ and the transformation to generate the exported metric.
 |:---|:---|
 | **Details** | Sum bytes for all traffic per network service | 
 | **Usage** | Evaluate network usage breakdown per network service | 
-| **Labels** | bandwidth, graph, rate, network-service |
-| **Operation** | aggregate by `service` and `sum` field `bytes` |
+| **Tags** | bandwidth, graph, rate, network-service |
+| **Operation** | aggregate by `service, _RecordType` and `sum` field `bytes` |
```

Comment:
Is _RecordType specified anywhere in the documentation?

Reply:
not yet